tasks/release_osx.js

```diff
@@ -79,6 +80,17 @@ var renameApp = function() {
     return Q();
 }
 
+function signApp() {
```

Comment:
1. Please keep consisten indentation style (4 spaces) with the rest of the project.
2. Also I use in this project form `var name = function () {}`, not `function name() {}`
Besides that it's awesome!